release_notes/44-release-notes.adoc

```diff
@@ -645,6 +646,33 @@ The following changes are included:
 
 * This release updates the process to ensure that Sensor completes the TLS handshake during secret synchronization. If this does not happen, the process can freeze and leave Sensor with incomplete data stores and a corrupted state, even if Sensor appears healthy.
 
+[id="resolved-in-version-445_{context}"]
+=== Resolved in version 4.4.5
+
+*Release date*: 29 August 2024
+
+This release of {product-title-short} fixes the following bugs:
+
+* Fixed an issue where Sensor could not identify pull secrets because the annotation was changed from `kubernetes.io service-account.name` to `openshift.io/internal-registry-auth-token.service-account` in the {ocp}.
+
+* Fixed an issue where policies incorrectly displayed an enforced value of `Yes` in the *Configuration Management -> Application & Infrastructure -> Deployments* page due to inconsistent status handling, while showing `No` in the *Violations* page.
+
+* Fixed an issue where outdated `nghttp2` libraries in various container images had vulnerabilities due to old base images.
+
+* Fixed an issue where outdated `libuv` versions in various container images had vulnerabilities due to the use of old base images.
+
+* Fixed an issue where outdated `python-idna` libraries in various container images had vulnerabilities due to old base images.
```

Comment:
Are these necessary if we talk about fixed vulns later? Though, it doesn't seem like we reference which CVEs were resolved for these packages

Reply:
We can remove if this is not needed. 
I documented these based on the following tickets:
- https://issues.redhat.com/browse/ROX-25095
- https://issues.redhat.com/browse/ROX-25096
- https://issues.redhat.com/browse/ROX-25097